docs/source/api.rst

```diff
@@ -433,7 +458,7 @@ Specify whether TCP keep-alive should be enabled.
 :Type: ``bool``
 :Default: ``True``
 
-**This is experimental.** (See :ref:`filter-warnings-ref`)
+**This is experimental** (see :ref:`filter-warnings-ref`).
```

Comment:
Should this be preview?

Reply:
See https://github.com/neo4j/neo4j-python-driver/pull/890/#discussion_r1162570383